docs/ipdk-security.md

```diff
@@ -60,44 +59,54 @@ InfraP4D will check for certificate in the following default location:
 
 If alternate location is desired, the location will need to be specified during runtime with the following flags
 
-> -ca_cert_file=[CA certificate file]
-> -server_cert_file=[Server certificate file]
-> -server_key_file=[Server private key file]
-
-For e.g.:
+```text
+-ca_cert_file=[CA certificate file]
+-server_cert_file=[Server certificate file]
+-server_key_file=[Server private key file]
+```
+For example:
 
 ```bash
 # Files present in /tmp/certs/ directory
- 
 $IPDK_RECIPE/install/sbin/infrap4d  -ca_cert_file=/tmp/certs/ca.crt  -server_cert_file=/tmp/certs/stratum.crt  -server_key_file=/tmp/certs/stratum.key
+```
 
+### Client certificate verification
+
+InfraP4D requires connecting gRPC clients to send a valid certificate that can be verified. A flag is available to the users to tune the level of security required. The available values are:
+
+```text
+NO_REQUEST_CLIENT_CERT
+REQUEST_CLIENT_CERT_NO_VERIFY
+REQUEST_CLIENT_CERT_AND_VERIFY
+REQUIRE_CLIENT_CERT_NO_VERIFY
+REQUIRE_CLIENT_CERT_AND_VERIFY (default)
 ```
+More info on these values can be found on [this gRPC library documentation page](https://grpc.github.io/grpc/cpp/grpc__security__constants_8h.html#a29ffe63a8bb3b4945ecab42d82758f09).
```

Comment:
blank line between second row of backticks and following paragraph

I believe Markdown calls these "fences". You put them around (before and after) headings and blocks of display text

Reply:
Fixed